src/transform/plugins/imsize/plugin.ts

```diff
@@ -206,6 +206,27 @@ export const imageWithSize = (md: MarkdownIt): ParserInline.RuleInline => {
             if (height !== '') {
                 token.attrs.push([ImsizeAttr.Height, height]);
```

Comment:
Why we still need height and width attributes?

Reply:
https://github.com/diplodoc-platform/transform/blob/bff7df970b50381326c0392bcf7e2347d3cd9bec/src/transform/plugins/changelog/index.ts#L78
Alas, that's why we still need it

Reply:
I also noticed complaints about incorrect image appearance when the height was not proportional. However, I did not see any requests for a stretched image or an image with extra frames.

Reply:
The proportions of the image are calculated based on the height and width specified in the markup. If the container is larger than these dimensions, the image will maintain its original height and width. If the width of the container is less than the width of the image, the image's width will adjust to 100% of the container's width and the height will be adjusted proportionally based on the original height and width values from the markup.